benchalerts/benchalerts/pipeline_steps/conbench.py

```diff
@@ -97,8 +95,8 @@ def _get_one_run_comparison(self, run_id: str) -> Optional[RunComparisonInfo]:
         """
         try:
             contender_info = self.conbench_client.get(f"/runs/{run_id}/")
-        except requests.HTTPError as e:
-            if e.response.status_code == 404:
+        except RetryingHTTPClientNonRetryableResponse as e:
+            if e.error_response.status_code == 404:
```

Comment:
Thank goodness for a 98%+ coverage test suite.

Reply:
Oops, I hope my comment didn't come off as snarky! I was trying to convey my relief that my time spent on unit tests in the past had paid off 🙂

Reply:
> I hope my comment didn't come off as snarky

Not at all!